src/Card/index.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import "./Card.scss";
+
+const Card = props => {
+  const {
+    image,
+    message,
+    receivedTime,
+    imagePosition,
+    classNamePrefix,
+    detailPage,
+    cardOptions,
+    renderImage
+  } = props;
```

Comment:
you can destructure like 
`const Card = ({image,
    message,
    receivedTime,
    imagePosition,
    classNamePrefix,
    detailPage,
    cardOptions,
    renderImage}) => {
}`